xbmc/pvr/timers/PVRTimerInfoTag.h

```diff
@@ -64,6 +64,20 @@ namespace PVR
   typedef boost::shared_ptr<PVR::CPVRTimerInfoTag> CPVRTimerInfoTagPtr;
   #define PVR_VIRTUAL_CHANNEL_UID (-1)
 
+  enum WeekdayMask
```

Comment:
Shouldn't these be defined in `xbmc_pvr_types.h` as well since the `iWeekDays` integer is a part of the timer struct? Otherwise the addons will have to rely on magic values.